aptos-move/framework/aptos-framework/sources/voting.move

```diff
@@ -281,18 +296,6 @@ module aptos_framework::voting {
         option::extract(&mut proposal.execution_content)
     }
 
-    /// Return true if the voting on the given proposal has already concluded.
-    /// This would be the case if the proposal's expiration time has passed or if the early resolution threshold
-    /// (if specified) has been reached.
-    ///
-    /// @param voting_forum_address The address of the forum where the proposals are stored.
-    /// @param proposal_id The proposal id.
-    public fun is_voting_closed<ProposalType: store>(voting_forum_address: address, proposal_id: u64): bool acquires VotingForum {
-        let voting_forum = borrow_global_mut<VotingForum<ProposalType>>(voting_forum_address);
-        let proposal = table::borrow_mut(&mut voting_forum.proposals, proposal_id);
-        can_be_resolved_early(proposal) || timestamp::now_seconds() >= proposal.expiration_secs
-    }
-
```

Comment:
you cannot delete public methods. the best you can do is abort

Reply:
Reverted. This was an unnecessary refactor.